### DISCUSSION TITLE
List of research questions

### DISCUSSION THREAD

**Discussion by mattosborn** [life-itself/community#591] [Research] [answered]
I’m curious if we have (or if not, could compile) a list of open research questions relevant to the focus topics in this community.

Something along the lines of:
https://80000hours.org/articles/research-questions-by-discipline/

### Why?
1. Help facilitate and deepen general alignment within the community.
2. Collective ideation and inspiration around research topics and projects. It feels like there’s a lot of siloed knowledge, ideas and thought that may benefit and resonate with others.
3. Attempt to spark discussion and generate traction around specific research questions
4. Links the abstract _topic map_ and _research community overview page_ into concrete questions and tangible research projects.
5. Help newcomers more quickly get an overview of our specific interests (figure out whether they are aligned)

Assuming we don't have this already and people like the idea, I'd be happy to gather, compile and publish collected contributions.

**Comment by rufuspollock** [comment]
This is a great idea @mattosborn 👏👍

Can involve both compiling what we already have and collecting new ideas and suggestions.

In terms of existing stuff main stuff from 2016-2019 efforts are here https://github.com/life-itself/community/issues/96 (easiest thing here would be to publish that stuff!)

**Reply by mattosborn** [reply]
I had a look through and the main relevant documents I could find were:

- [Research Program - README 2017](https://docs.google.com/document/d/18_vdpVcpKCvdhpwO3cUDPBsMttMD9AzueH4e7cVjHhc/edit)
and
- [Culture and Values](https://docs.google.com/document/d/1NIYkPh40ZSup5zmbm5x0qDodWZz6uB055YY-1E5QP60/edit)

LMK if you think I missed something there.

There's not a ton of actual focused research questions between them but I'll see if I can pull some out. A lot of these look more like explainers, general projects, or [distillation](https://distill.pub/2017/research-debt/) as opposed to original research questions, and it's probably helpful to distinguish between those rather than bundling them together.

**Reply by rufuspollock** [reply]
@mattosborn yes, i would say our research program is more humble at the moment in terms of compiling what already exists and then coming to novel work.

The most relevant image is probably this one from that work:

![image](https://github.com/life-itself/community/assets/180658/c668f856-9125-40f3-bd68-df1e95ed82c0)

**Comment by rufuspollock** [comment, marked as answer]
Here's a first pass on a high level answer to the question of "What are we focused on in our research program" and what are our major research questions.

## Summary: Key research question

- **Domains/Lines**: What are the different lines or domains of human development e.g. cognitive complexity and morality could be two distinct domains.
- **Maps**: What are the "maps" of each of those domains e.g. what are the different locations or levels you can be in in that domains
- **Rafts**: What are the factors that are associated with movement around a domain? Specifically, what are the "rafts" that you can use to move around e.g. practices, environments etc. 

## How do we arrive to these?

### 1. What are the paths to an awakening society?

At the top level, as we write on the [research page](https://lifeitself.org/research):

> Exploring paths to an awakening society -- or, a wiser, weller world.

NB: Going forward, we can call it awakening society or a wiser world.

### Specifically: 2.1. What is an awakening society 2.2. How do we get there? 2.3 Why do we want to go there?

So, the next level down is the immediate obvious sub-questions like:

- First, what is a wiser, weller world? What does an awakening society look like?
- Second, how do you get there?

Or as in the the metaphor we used in the original intro to the research collective:

> What mountains are we going to ... and how do we climb them?

### Awakening societies involve much greater focus on human development

Now that is very general. And we have a view already about some of that i.e. about the mountain we're going to and how to get up it. So isn't a completely open-ended question.

Our thesis is that wiser, weller worlds involve a much greater prioritization of inner development.

### What is human development and how does it happen?

 So central to a research program is: the question what is human development and how does it happen?

More formally we can term these topics ontology and culturology i.e. the study of "being" and of culture ("collective being"). We could even more specifically say ontogenetics i.e. the development of being. Similarly, we could say culturo-genetics or cultural evolution. Ontogenetics can sound a bit weird but it was actually exactly the term being used by e.g. James Mark Baldwin in early developmental psychology in late 19th century.

We could use the term psychology but it has become a bit narrowly focused on the mind and cognition. Hence we prefer ontology or ontogenesis as we're interested in all the aspects of being for human beings and how they can develop.

<table>
<tr>
 <td>Personal</td>
 <td>Collective</td>
<tr>
 <td>Ontology and Ontogenesis</td>
 <td>Culturology and Culturo-genesis or Cultural Evolution</td>
</table>

### What is human development? And how does it happen? What factors contribute to it?

And so a big research question would be that: 

> What is human development?  And how does it come about? What factors contribute to it?

Again an aside re terminology: some people don't like the term development because it can imply some kind of hierarchy. We'll come back to the maps terminology below which is more neutral. But for now we use development because that's what the field has generally used.

### Development is multi-dimensional or multi-domain

The next point is that there's general agreement that there isn't a single line of development for human beings. Instead we can develop along multiple different "lines" or domains of development and at different speeds.

A simple demonstration of this is that cognitive and moral development may be different: you can find people who are smart and are not very nice (e.g. some psychopaths).

### Qu: what are the major developmental domains?  

So one thing that would be in research program would be:

> What are these domains?

That's a really big question. I note that you may answer this question iteratively in tandem with the mapping and rafting i'll talk about next i.e. working out domains may only become clear over time.

### Maps and rafts: 3.1) what are the maps of given domains? 3.2) what are the rafts that allow us to move around a given map?

Then for each domain we want:

- A map shows the locations or levels within that domain? what places can i go to in that domain
- What are the rafts? i.e. the tools or activities that allow me to move around the map in those domains.

## Examples and commentary

### Example domain: moral development

One version of morality that's covered right back in Plato's Gorgias is: might is right. What's right is what the powerful to do. It can even be more ego-centric: what's good is what I want.

Then another location in morality may be: It's about following the rules. what's good is following the rule. 

But then at some point you might come to question that. is it possible for the laws to be immoral? Very famously and a classic example was: could you try Nazis for obeying their orders to do something like killing Jews. Are they protected by the fact that they were just following orders? for example?

And there's a sense for many people that there's a deeper level of morality. A morality beyond the rules of a given society and, so, yes what the Nazis did was wrong even if they were obeying orders. It goes beyond to the conventional to some kind of universal, maybe.

And then we have general principles like it's about maximizing utility like we're gonna base morality on utility optimization, doing the greatest good for the greatest number or whatever.

And there you have the start of a map for that domain.

And it's also the case that most humans, at least as children, develop through those to some extent.

So very, very young children, tend to think about themselves. They don't have in them, a lot of conception of the other, other people's needs.

And then at some point there is this sense of power: don't do it because Daddy's gonna tell you off. And they respect that. That's just like there this person who's bigger and more powerful than me. Just like mummy is.

This is X the stronger than them. They don't really feel it's unfair necessarily.

Then there's a point when it's like, man, it's the rules. What is it my parents say, what is it my school says etc.

What I'm telling you here is a bit about Lawrence Kohlberg's work who was the student of Piaget.

Piaget was this developmental psychologist and Kohlberg was his student. Piaget already had quite a bit on this. But Kohlberg did a lot of studies starting in the 50s of this and had this theory of how these stages unfolded.

I emphasize that whilst there may be this unfolding or trajectory there might just be these different locations without one being before or after another.

And i mention that for many many people in the world today don't actually get much beyond conventional morality. They don't get beyond: following the rules is what makes you a good person.

And a bit question is how do people go from the rules to some sense of there being a deeper more universal truth?

### We are focused on synthesizing

Right now, we are primarily in the business of synthesizing rather than doing novel work. Now, I think that may lead to other things, but right now doing those kind of syntheses is the priority. 

### Example Domain: cognitive complexity

So you can pick other domain which relates the kind of complexity of thought.

A classic example would be going from single-variate linear thinking to multivariate feedback thinking.

**Example**: planting pine trees for a forest for later harvesting. (hat-tip to Suzanne Simard for this example)

In a linear thinking type model: I apply pesticide in a forest to kill off all the other trees so they don't compete with my trees. So I'm spraying everything with Roundup.

And then it turns out that actually there's a dyadic ecosystem interaction between the pine trees and the birch trees so that when the birch trees grow they actually draw in nutrients and they give them to the pine tree later and there's a kind of more symbiotic model. And by killing off the birch trees you are actually harming the growth of the pine trees, believe it or not.

So the first model would be a bit more linear. Like I do X and get Y. and the second is a more multi-variate model.

**Reply by mattosborn** [reply]
This is great and is exactly the kind of clarity I was looking for, so thank you 👍

It raises a bunch more questions for me. One of them is around how you arrive at the key research questions. They seem like a very small piece of a much bigger puzzle, so I'm trying to understand exactly where they fit. It seems like there are some logical jumps required to get to them that aren't covered in your post.

I guess my query here boils down to the following:

1. How will the answers to these questions be used, and by whom?
2. How deep of an understanding of Human Development and its related practices do we need before we can start charting a path towards a society that embraces it?
3. Is our current understanding of Human Development a limiting factor here?
4. Is it possible that our understanding of Human Development is already advanced enough?

There's a similarity to the Primacy of Being view here. Much like with climate change, people can develop ever more complex and accurate theories and practical solutions, but at some point, if those solutions are going to be of any use, you need people to actually start doing the thing. Is deepening our understanding of Human Development a prerequisite to us "doing the thing"?

**Reply by rufuspollock** [reply]
You are quite right this is only one set of research questions however it is, i think, both the "spine"of our research program "skeleton" and a natural starting point.

### "Spine"

This argument was approximately set out above: human development plays a central role both in defining a wiser, weller world *and* in getting us there.

### Starting point

I posted up a very simplified "theory of change" a while back:

https://lifeitself.org/notes/strategy-macro-to-micro-2023

Recently i've expanded to recurse further and you end up with research playing a central role and within research the maps and rafts stuff:

![image](https://github.com/life-itself/community/assets/180658/b5ca29e1-18b5-4221-a4d9-6bf815e324a6)

### Meta-capacity

In our own efforts to collaborate and engage we are going to needs these developmental capacities.

> This is kind of a bonus.

## Answering your specific questions 

- How will the answers to these questions be used, and by whom? **I think the potential user set is broad from those designing, evaluating etc deliberately developmental programs, those creating conscious communities, movements builders creating 2nd renaissance/ontological political programs etc**
- How deep of an understanding of Human Development and its related practices do we need before we can start charting a path towards a society that embraces it? **I think we can dance back and forth between theory and praxis**
- Is our current understanding of Human Development a limiting factor here? **Yes, i've come to think it is. The example of [Martin and his work on waking up](https://lifeitself.org/blog/2023/01/19/a-scientific-approach-to-awakening-and-fundamental-wellbeing) makes me think that better maps with associated evaluations is essential underpinning to levelling up rafts etc. If you don't know what you are measuring or where you are going you can't really improve. Right now there's just a zoo of different techniques being tried out in god knows what combinations and we just don't know what is happening. Martin probably only just $1-2m a year but he was focused and persistent.**
- Is it possible that our understanding of Human Development is already advanced enough? **So far, I don't think we have decent maps yet in most areas, at least to my knowledge. e.g. the "growing up" dimension is dominated by the spiral dynamics color spectrum (at least in integral community) yet there's an incredible paucity of evidence. our studies on moral development a la Kohlberg seem to number under a 1000 actual evaluated cases etc.**

> There's a similarity to the Primacy of Being view here. Much like with climate change, people can develop ever more complex and accurate theories and practical solutions, but at some point, if those solutions are going to be of any use, you need people to actually start doing the thing. Is deepening our understanding of Human Development a prerequisite to us "doing the thing"?

- We need to dance between theory and praxis with a strong emphasis on the praxis.
- And ... we waste incredible amounts of time when we go in the wrong direction. Again I can't emphasize how powerful i find the Martin example with waking up ...
- We don't need to spend decades here
- And matters are so urgent we can afford to be patient 😉
